src/Microsoft.Net.Http.Headers/CookieHeaderValue.cs

```diff
@@ -39,7 +39,7 @@ public string Name
             get { return _name; }
             set
             {
-                CheckNameFormat(value, "name");
```

Comment:
@rtpHarry I replaced the string literal to the original backend field after I saw that line of code